graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,353 @@
+            "E-step".
+
+        iterative_num_reps : int (default=100), optional
+            Number of reps in each iteration of the iterative optimal transport
+            problem. I.e. maxumum number of total iterations the whole "EM"
+            algorithm.
+
+        init : string, {'2d' (default), 'sign_flips', 'custom'}, optional
+
+            - '2d'
+                Uses :math:`2^d` different restarts, where :math:`d` is the
+                dimension of the datasets. In particular, tries all matrices
+                that are simultaneously diagonal and orthogonal. In other
+                words, these are diagonal matrices with all entries on the
+                diagonal being either +1 or -1. The final result is picked
+                based on the final values of the objective function. This is
+                motivated by the fact that in the graph setting, the embeddings
+                have two types of orthogonal non-identifiability, one of which
+                is captured by the orthogonal diagonal matrices. For more on
+                this, see [2]_.
+            - 'sign_flips'
+                Initial alignment done by making the median value in each
+                dimension have the same sign". The motivation is similar to
+                that in '2d', except this is a heuristic that can save time,
+                but can sometimes yield suboptimal results.
+            - 'custom'
+                Expects either an initial guess for :math:`Q` or an initial
+                guess for :math:`P`, but not both. See `initial_Q` and
+                initial_P`, respectively. If neither is provided, initializes
+                `initial_Q` to an identity with an appropriate number of
+                dimensions.
+
+        initial_Q : np.ndarray, shape (d, d) or None, optional (default=None)
+            An initial guess for the alignment matrix, Q, if such exists. Only
+            one of initial_Q, initial_P can be provided at the same time, and
+            only if `init` argument is set to 'custom'. If None, and initial_P
+            is also None - initializes `initial_Q` to identity matrix. Must be
+            an orthogonal matrix, if provided.
+
+        initial_P : np.ndarray, shape (n, m) or None, optional (default=None)
+            Initial guess for the optimal transport matrix, P, if such exists.
+            Only one of initial_Q, initial_P can be provided at the same time,
+            and only if `init` argument is set to 'custom'. If None, and
+            initial_P is also None - initializes `initial_Q` to identity
+            matrix. Must be a doubly stochastic matrix if provided (rows sum up
+            to 1/n, cols sum up to 1/m.)
+
+    Attributes
+    ----------
+        Q_ : array, size (d, d)
+            Final orthogonal matrix, used to modify X.
+
+        P_ : array, size (n, m) where n and md are the sizes of two datasets
+            Final matrix of optimal transports
```

Comment:
Explain what these are in simple terms, i think of it as "soft matching weights from points in one dataset to the other" but maybe there's a better way to say?

Reply:
I think of them in the same way.

Reply:
added and clarified normalization.